### PR TITLE
[FIX] detect version by existing members

### PR DESCRIPTION
**Pull request by hbrunn**
as I ran into pip version issues in #7, I suggest to explicitly test all ubuntu LTS versions, this covers all pip versions we want to support. With this I think I'm then also more comfortable submitting #5 and I'll rebase #7 then

### CODE REVIEW

**Comment by hbrunn** [comment]
it bothers me that the tests seem to take forever, that wasn't the case at the weekend. Temporary fluke?

**Comment by StefanRijnhart** [comment]
Then again, we don't have that much contributions here do we.

**Review by StefanRijnhart — APPROVED** [review]
Thanks for doing this tedious but essential work!

**Comment by StefanRijnhart** [comment]
Mmm, 19 hours is really outside of normal queuing I hope. Restarted the tests.

image

File: .github/workflows/tests-with-odoo.yml [hunk 1]
```diff
@@ -8,11 +8,12 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-${{ matrix.ubuntu_version }}
     strategy:
       fail-fast: false
       matrix:
         odoo_release: ["12.0", "13.0", "14.0", "15.0"]
+        ubuntu_version: ["bionic", "focal", "latest"]
```
Comment (StefanRijnhart) [inline .github/workflows/tests-with-odoo.yml:16]: Maybe the nicknames are not recognized. See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources. And that document indicates that latest is a synonym for 20.04, not 22.04 :confused: 
```suggestion
        ubuntu_version: ["18.04", "20.04", "22.04"]
```

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**